Xamarin.PropertyEditing.Mac/Controls/EntryPropertyEditor.cs

```diff
@@ -21,10 +21,9 @@ public EntryPropertyEditor (IHostResourceProvider hostResources)
 
 			RightEdgeConstraint = NSLayoutConstraint.Create (Entry, NSLayoutAttribute.Right, NSLayoutRelation.Equal, this, NSLayoutAttribute.Right, 1f, 0);
 			AddConstraints (new[] {
-				NSLayoutConstraint.Create (Entry, NSLayoutAttribute.CenterY, NSLayoutRelation.Equal, this,  NSLayoutAttribute.CenterY, 1f, 0),
```

Comment:
Because if it is `CentreY` constrained, when the control height changes it will position itself nicely in the horizontal centre. Like this...
<img width="320" alt="Screen Shot 2019-07-15 at 18 29 18" src="https://user-images.githubusercontent.com/271363/61235968-bd6ecc00-a72e-11e9-9f11-a55f5978b415.png">

The current `Bottom` constrain looks like this..
<img width="316" alt="Screen Shot 2019-07-15 at 18 30 12" src="https://user-images.githubusercontent.com/271363/61236029-ed1dd400-a72e-11e9-970c-89071fbffe62.png">

I didn't like the idea of constraining it to the Variation label. Open to any other suggestions.

Reply:
Even if this was the correct change (not sure), you only applied it to one editor, but most have this same layout. I'll see if I can come up an alternative.